src/pages/editor-page/canvas/canvas.tsx

```diff
@@ -31,6 +31,8 @@ const initialEdges: TableEdgeType[] = [];
 export interface CanvasProps {}
 
 export const Canvas: React.FC<CanvasProps> = () => {
+    const isMobile = window.innerWidth <= 767;
```

Comment:
```suggestion
    const { isMd: isDesktop } = useBreakpoint('md');
    const isMobile = !isDesktop;
```
You can use useBreakpoint instead, like we do in the editor-page.tsx component.